### PR TITLE
Feature/test modules  endpointform

### PR DESCRIPTION
**Pull request by ivanzusko**

### CODE REVIEW

File: src/__tests__/modules/EndpointForm/EndpointForm.test.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+import { Route, Link, MemoryRouter } from 'react-router-dom';
+
+import apiSchema from '../../../configurations/apiSchema.config';
+import { renderFakeForm, wrap } from '../../../utils/createTestForm';
+
+import EndpointForm from '../../../modules/forms/EndpointForm/EndpointForm';
+
+const initialValues = {
+    proxy: {
+        upstreams: 'sdeded'
```
Comment (s4nji) [inline src/__tests__/modules/EndpointForm/EndpointForm.test.js]: Maybe a clearer name (`mock-upstream`) or use random string 😉

File: src/__tests__/modules/EndpointForm/EndpointForm.test.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import { createStore } from 'redux';
+import { mount } from 'enzyme';
+import toJSON from 'enzyme-to-json';
+import { Route, Link, MemoryRouter } from 'react-router-dom';
+
+import apiSchema from '../../../configurations/apiSchema.config';
+import { renderFakeForm, wrap } from '../../../utils/createTestForm';
```
Comment (s4nji) [inline src/__tests__/modules/EndpointForm/EndpointForm.test.js]: `wrap` is not used, can be removed

File: src/__tests__/modules/EndpointForm/EndpointForm.test.js [hunk 3]
```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import { createStore } from 'redux';
+import { mount } from 'enzyme';
+import toJSON from 'enzyme-to-json';
```
Comment (s4nji) [inline src/__tests__/modules/EndpointForm/EndpointForm.test.js]: Also not used, can be removed

File: src/__tests__/modules/EndpointForm/EndpointForm.test.js [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+import { Route, Link, MemoryRouter } from 'react-router-dom';
```
Comment (s4nji) [inline src/__tests__/modules/EndpointForm/EndpointForm.test.js]: And this `Link`

**Comment by ivanzusko** [comment]
Thanx, @s4nji !!! 👍

File: src/__tests__/modules/EndpointForm/EndpointForm.test.js (hunk 4 above)
Comment (ivanzusko) [reply src/__tests__/modules/EndpointForm/EndpointForm.test.js]: And `Router` 😄

**Pull request merged**